### DISCUSSION TITLE
What is this request? GET /api/.env is someone trying to hack my database??

### DISCUSSION THREAD

**Discussion by idea2547** [pocketbase/pocketbase#4127] [Q&A]
![Screenshot_2024-01-11-16-22-32-95_40deb401b9ffe8e1df2f1cc5ba480b12](https://github.com/pocketbase/pocketbase/assets/53974956/11f89a38-c59c-4a29-a171-f55dbb26487a)

**Comment by jonshipman** [comment]
Maybe? Hard to tell without knowing the stacks you're using. As long as you don't host a .env route in pocketbase, it's just usual web chatter.

**Comment by physicsSorcererKing** [comment]
You get this request because it's online.
